app/controllers/concerns/hyrax/works_controller_behavior.rb

```diff
@@ -437,5 +439,21 @@ def concern_has_file_sets?
     def uploaded_files
       UploadedFile.find(params.fetch(:uploaded_files, []))
     end
+
+    def available_admin_sets
```

Comment:
It seems like the pattern in the controllers is to store methods that do this sort of work inline. Should we consider a controller helper?

Reply:
can you say more? i'm not sure i'm understanding the suggestion.

Reply:
Sorry about that. At best, it's probably suggesting a premature optimization, and initially I was reading the wrong file location..

I guess in short, in this controller concern in particular, there are quite a few methods doing various things.

So I'm wondering if it makes sense to eventually extract this to its own concern/class. Something like that.